LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -236,10 +236,13 @@
     "web-platform-tests/html/browsers/windows/targeting-cross-origin-nested-browsing-contexts.html": "skip",
     "web-platform-tests/html/canvas": "import",
     "web-platform-tests/html/canvas/element": "import",
+    "web-platform-tests/html/canvas/element/fill-and-stroke-styles": "import",
     "web-platform-tests/html/canvas/element/wide-gamut-canvas": "import",
     "web-platform-tests/html/canvas/offscreen": "import",
+    "web-platform-tests/html/canvas/offscreen/fill-and-stroke-styles": "import",
     "web-platform-tests/html/canvas/resources": "import",
     "web-platform-tests/html/canvas/tools": "import",
+    "web-platform-tests/html/canvas/tools/yaml-new/fill-and-stroke-styles.yaml": "import",
```

Comment:
Can we avoid adding these redundant entries?

Reply:
It would be nice if the script did the right thing, but I can manually take them out again.